.travis.yml

```diff
@@ -28,6 +28,7 @@ before_install:
 install:
 - python devtools/scripts/create_conda_env.py -n=test -p=$PYTHON_VER devtools/conda-envs/test_env.yaml
 - conda activate test
+- python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
```

Comment:
Sorry for the late review -- It might be good to add a comment here explaining the purpose of this line.